### PR TITLE
Encode tiles using Base64

### PR DESCRIPTION
**Pull request by jmorton**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -52,6 +52,7 @@
                  ;; Data types, encoding, etc.
                  [byte-streams "0.2.0"]
                  [clj-time "0.11.0"]
+                 [commons-codec "1.9"]
```
Comment (oubiwann) [inline project.clj]: Did you try using the native Clojure library? (https://github.com/clojure/data.codec)

That's what I've used in the past ...

Comment (jmorton) [inline project.clj]: Do you mean... 

https://github.com/clojure/data.codec

? ...can do, looks like it fits the bill even though it only implements base64. If there is another call built into Clojure core that's escaping me, let me know. I know Java 8 finally ads base64 encoding but think we are targeting Java 7. 

;j-

**Comment by oubiwann** [comment]
We chatted in person about this. We'll just use Apache Commons w/o the wrapper.

There is a reference or two to the Clojure base64 wrapper for Apache Commons in the code base already -- we should remove that ASAP.

**Comment by oubiwann** [comment]
Added clean-uo ticket #5 to address the outstanding bits here.

**Comment by oubiwann** [comment]
Thanks, Jon! Merging ...

**Pull request merged**